### ISSUE TITLE
Create stand-alone event data download page

### ISSUE THREAD

**Issue by nathanmwilliams** [townhallproject/admin#103]
http://thp-admin.herokuapp.com/federal-events (or something to that effect)

-Button to download .csv of all live federal event data.

-Do not include "Entered_By" email address field.

-Do include "Chamber" field to identify House vs Senate

**Comment by meganrm** [comment]
is this for partners to download events? is that why you dont just want it on the 'live' events tab?

**Comment by nathanmwilliams** [comment]
Yeah, for data partners that just want to download a daily/weekly csv of events.
